### PR TITLE
Copy postgres log files for noobaa/mcg

### PR DESCRIPTION
**Pull request by alphaprinz**
For the noobaa's postgres pod, also copy the log directory.

### CODE REVIEW

**Comment by black-dragon74** [comment]
LGTM

**Review by crombus — APPROVED** [review]

File: must-gather/collection-scripts/gather_noobaa_resources [hunk 1]
```diff
@@ -68,6 +68,7 @@ NOOBAA_POSTGRES_LABEL='noobaa-db in (postgres)'
 for pod in $(oc -n "${ns}" get pods -l "${NOOBAA_POSTGRES_LABEL}" | grep -v NAME | awk '{print $1}'); do
     dbglog "collecting noobaa db pod logs from ${ns}"
     { timeout 120 oc -n "${ns}" logs --all-containers "${pod}" &>"${LOG_DIR}"/"${pod}".log; } >>"${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
+    { timeout 120 oc -n "${ns}" cp "${pod}":/var/lib/pgsql/data/userdata/log "${LOG_DIR}"/"${pod}"/pg_log; } >>"${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
```
Comment (agarwal-mudit) [inline must-gather/collection-scripts/gather_noobaa_resources:71]: @yati1998 Do we use cp command to copy the logs? What's the best way to do this?

Comment (yati1998) [reply must-gather/collection-scripts/gather_noobaa_resources:71]: We haven't used that before so not sure if it works. @alphaprinz did you try collecting the logs and see if it works? If not you can use oc rsync to copy files from the pod to the local directory.

Comment (alphaprinz) [reply must-gather/collection-scripts/gather_noobaa_resources:71]: It worked ok in my manualy test.
I should note that these are regular files and not text printed to standard output (so their content is not captured by the "oc logs" command).

**Comment by yati1998** [comment]
@alphaprinz please resolve the merge conflicts, I will then merge the PR.

**Comment by alphaprinz** [comment]
@yati1998 I've rebased and push to my forked branch.
BTW I tried using "make" after the rebase but I get-

vendor/github.com/openshift/build-machinery-go/make/targets/openshift/imagebuilder.mk:19: *** imagebuilder not found! Get it with: `go get [github.com/openshift/imagebuilder/cmd/imagebuilder@v1.2.1]
(http://github.com/openshift/imagebuilder/cmd/imagebuilder@v1.2.1)`.  Stop.

(Issuing the specified "go get" command doesn't help).

**Comment by black-dragon74** [comment]
> (Issuing the specified "go get" command doesn't help).

The makefile is for PROW use. To build locally please use `docker build . -t <your_tag>`

And then run it using `docker run <your_tag>`

Regards

**Comment by alphaprinz** [comment]
@yati1998 Hi,
Can you please tell me what are next steps?
Shoud I use "/retest"?

The "check-must-gather-binary" has this error log-
ERRO[2023-10-04T07:51:14Z] 
  * could not run steps: step root failed: error occurred handling build root-amd64: the build root-amd64 failed after 9s with reason InvalidContextDirectory: The supplied context directory does not exist. 

Thanks

**Review by yati1998 — APPROVED** [review]

**Pull request merged**